packages/fluentui/react-northstar/src/themes/teams/components/Dropdown/dropdownStyles.ts

```diff
@@ -149,6 +150,9 @@ export const dropdownStyles: ComponentSlotStylesPrepared<DropdownStylesProps, Dr
 
   triggerButton: ({ props: p, variables: v }): ICSSInJSStyle => {
     return {
+      marginBottom: pxToRem(-4),
```

Comment:
Do we want there to be a small unclickable area in that button since it is a part of the dropdown ? without the margin the button height will extend the height of the parent container

Reply:
So that is tricky, because what I am trying to achieve is to have less visual regression as possible but if we gonna say that having small regression is fine since we are changing the DOM I would be ok with removing this

Reply:
Yeah this isn't a show stopper for me... it's quite a minor thing I noticed playing with it. I'm not entirely sure about the regression scenario so you could ping someone who's been around longer about that.

Other than that looks good to me, nice catch on the flex-wrap :)